rspec-rails.gemspec

```diff
@@ -33,20 +33,24 @@ Gem::Specification.new do |s|
     s.cert_chain = [File.expand_path('~/.gem/rspec-gem-public_cert.pem')]
   end
 
-  version_string = ['>= 3.0']
-
-  if RUBY_VERSION <= '1.8.7' && ENV['RAILS_VERSION'] != '3-2-stable'
-    version_string << '!= 3.2.22.1'
-  end
+  version_string = ['>= 4.2']
 
   s.add_runtime_dependency %q<activesupport>, version_string
   s.add_runtime_dependency %q<actionpack>,    version_string
   s.add_runtime_dependency %q<railties>,      version_string
+
+  # in these blocks expected_rspec_version is set up to track the released
+  # versions of RSpec. RSpec Rails does not have lock step versioning with the
+  # rest of RSpec after version 4.0.0, so this sets up the correct dev versions
+  # that we want. These will need to change from time to time as new RSpecs
+  # get released.
   %w[core expectations mocks support].each do |name|
     if RSpec::Rails::Version::STRING =~ /[a-zA-Z]+/ # prerelease builds
-      s.add_runtime_dependency "rspec-#{name}", "= #{RSpec::Rails::Version::STRING}"
+      expected_rspec_version = "3.9.0.pre"
```

Comment:
Shouldn't this be `4.0.0.pre`?

Reply:
no, RSpec rails 4 will target RSpec-* 3.x